appveyor.yml

```diff
@@ -0,0 +1,60 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.077.1
+      BRANCH:              master
+
+cache:
+
+init:
+  - git config --global core.autocrlf input
+
+build_script:
+  - cd c:/projects/
+  - ps: |
+        Start-FileDownload "http://downloads.dlang.org/releases/2.x/$Env:D_VERSION/dmd.$Env:D_VERSION.windows.7z" -FileName dmd2.7z
+        7z x dmd2.7z > $null
+        Start-FileDownload "http://downloads.dlang.org/other/dm857c.zip" -FileName dm857c.zip
+        7z x dm857c.zip > $null
+        Start-FileDownload "http://ftp.digitalmars.com/bup.zip" -FileName bup.zip
+        7z x bup.zip dm\bin\implib.exe > $null
+        Start-FileDownload "http://semitwist.com/download/app/dmd-localextras.7z" -FileName extras.7z
+        7z x extras.7z > $null
+        Start-FileDownload "http://nsis.sourceforge.net/mediawiki/images/c/c9/Inetc.zip" -FileName inetc.zip
+        7z x -o"c:\Program Files (x86)\NSIS" inetc.zip 
+
+  - call "c:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+  - set PATH=c:\projects\dmd2\windows\bin;c:\projects\dm\bin;%PATH%
+
+  - if not exist build\nul md nul
+  - set BASE=%TEMP%\create_dmd_release
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/dmd.git "%BASE%/dmd"
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/druntime.git "%BASE%/druntime"
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/phobos.git "%BASE%/phobos"
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/tools.git "%BASE%/tools"
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/dlang.org.git "%BASE%/dlang.org"
+  - git clone --depth 1 --branch %BRANCH% https://github.com/dlang/dub.git "%BASE%/dub"
```

Comment:
Could have been a loop and/or function calls to repeat in case of network failures, e.g. https://github.com/dlang/dmd/blob/master/.circleci/run.sh

Of course, this isn't needed for this initial version.

Reply:
I've kept that task for your installer helper script https://github.com/dlang/installer/pull/275 ;-)

Reply:
We don't need to clone the repos for the install script